install.sh

```diff
@@ -26,6 +26,7 @@ ARCH="${ARCH/armv8l/armv7l}"
 # BOOTSTRAP_PACKAGES cannot depend on crew_profile_base for their core operations (completion scripts are fine)
 BOOTSTRAP_PACKAGES="crew_mvdir pixz ca_certificates ruby openssl"
 [ -x /usr/bin/zstd ] || BOOTSTRAP_PACKAGES="zstd ${BOOTSTRAP_PACKAGES}" # use system zstd if available
+[ -x "${CREW_PREFIX}"/bin/xz ] && rm "${CREW_PREFIX}"/bin/xz # remove local xz if installed
```

Comment:
Why are we removing the local `xz`?

Reply:
We should abort the install with a message if there is anything installed in `CREW_PREFIX`.  Either that or offer to remove everything and continue.  It's possible other packages (think `rsync`) can break the install too.  This is too much of a hack.

Reply:
We're dealing with one file here. Maybe offering to remove everything should be part of the larger refactor effort in https://github.com/chromebrew/chromebrew/pull/7297 ?

I'm just trying to help people avoid a **_single_** point of failure they might encounter when installing over a previous install...

Reply:
`rsync` might also cause an issue, but I haven't encountered that as a problem in my multiple multiple installs and reinstalls.